docs/platforms/javascript/common/configuration/micro-frontend-support.mdx

```diff
@@ -37,61 +46,68 @@ module.exports = {
-        moduleMetadata: ({ org, project, release }) => {
-          return { team: "frontend", release };
-        },
+        moduleMetadata: ({ release }) => ({ dsn: "__MODULE_DSN__", release }),
       },
     }),
   ],
 };
 ```
 
-### `ModuleMetadata` Integration
-
-Requires SDK version `7.59.0` or higher.
-
 Once metadata has been injected into modules, the `moduleMetadataIntegration` 
 can be used to look up that metadata and attach it to stack frames with
-matching file names. This metadata is then available in other integrations or in
-the `beforeSend` callback as the `module_metadata` property on each
-`StackFrame`. This can be used to identify which bundles may be responsible
-for an error and used to tag or route events.
+matching file names. This metadata is then available in the `beforeSend` callback
+as the `module_metadata` property on each `StackFrame`. This can be used to identify
+which bundles may be responsible for an error and, once the destination is determined,
+store it as a list of DSN-release pairs in `event.extra[MULTIPLEXED_TRANSPORT_EXTRA_KEY]` 
+where multiplexed transport knows to look for.
+
+In practice here is what your Sentry intialization should look like:
 
 ```javascript
-import * as Sentry from "@sentry/browser";
+import { init, makeFetchTransport, moduleMetadataIntegration, makeSimpleMultiplexedTransport, MULTIPLEXED_TRANSPORT_EXTRA_KEY } from "@sentry/browser";
 
-Sentry.init({
-  dsn: "___PUBLIC_DSN___",
-  integrations: [Sentry.moduleMetadataIntegration()],
+init({
+  dsn: "__DEFAULT_DSN__",
+  integrations: [moduleMetadataIntegration()],
+  makeMultiplexedTransport(makeFetchTransport),
   beforeSend: (event) => {
-    const frames = event?.exception?.values?.[0].stacktrace.frames || [];
-    // Get all team names in the stack frames
-    const teams = frames
-      .filter((frame) => frame.module_metadata && frame.module_metadata.team)
-      .map((frame) => frame.module_metadata.team);
-    // If there are teams, add them as extra data to the event
-    if (teams.length > 0) {
-      event.extra = {
-        ...event.extra,
-        teams,
-      };
+    if (event?.exception?.values?.[0].stacktrace.frames) {
+      const frames = event.exception.values[0].stacktrace.frames;
+
+      // Find the last frame with module metadata containing a DSN
+      const route_to = frames
+        .filter((frame) => frame.module_metadata && frame.module_metadata.dsn)
+        .map((v) => v.module_metadata)
+        .slice(-1); // using top frame only - you may want to customize this according to your needs
+
+        event.extra = {
+          ...event.extra,
+          [MULTIPLEXED_TRANSPORT_EXTRA_KEY]: route_to,
+        };
     }
 
     return event;
   },
 });
+```
+
+Once this is set up, errors - both handled and unhandled - will be automatically routed to the right project. You don't need to do anything special.
 
+```
 Sentry.captureException(new Error("oh no!"));
+// or
+throw new Error("oops");
 ```
```

Comment:
I think we could remove this code sample since it's not really necessary. If you want to keep it, I'd recommend adding a line before to contextualize how users should use it. For example, "You can use add the following to your code to throw an error and double check that it's routed to the right project."

Reply:
done